.travis.yml

```diff
@@ -2,11 +2,10 @@ language: node_js
 
 # https://github.com/nodejs/LTS
 node_js:
+  - '10'
```

Comment:
latest version of eslint is no longer compatible with node@4 and @7 so remove them from CI, also added node@10. Those versions reached EOL so I think it's fine to drop them from CI.